app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -27,18 +29,22 @@ fun PartiesScreen(
 	navController: NavHostController,
 ) {
 	when (val state = viewModel.parties.value) {
-		is PartiesState.Loading -> {
+		is Loading -> {
 			PicoverLoader()
 		}
 
-		is PartiesState.Loaded -> {
+		is Loaded -> {
 			LoadedContent(
 				parties = state.parties,
 				openDetails = {
 					navController.navigate("partyDetails/$it")
 				},
 			)
 		}
+
+		is Error -> {
+			// TODO: Will be implemented in #133
```

Comment:
minor: You added here comment between `{ }` instead of doing the same as in `PartyDetailsScreen.kt` or vice versa

Reply:
unified :D (using this in brackets)